cea/tests/test_config.py

```diff
@@ -57,7 +57,8 @@ def test_applying_parameters(self):
 
     def test_decode_fileparameter(self):
         config = cea.config.Configuration()
-        expected_output = "/projects/heating/inputs/building-geometry/zone.shp"
+        scenario = config.general.scenario
+        expected_output = f"{scenario}/inputs/building-geometry/zone.shp"
```

Comment:
I tested this locally with `python -m unittest test_config.TestConfiguration` and passed all tests